### ISSUE TITLE
how did you convert data from .csv to gta.json

### ISSUE THREAD

**Issue by luc4leone** [bethrobson/CCC#1]
hi Elisabeth,

is it possible to see how did you convert [raw data](https://www.ncdc.noaa.gov/cag/global/time-series/globe/land_ocean/p12/12/1880-2017.csv) to gta.json?

**Comment by bethrobson** [comment]
Absolutely, I've uploaded the code into the repo. I was going to make it part of the course, but realized quickly it was getting too complex.

https://github.com/bethrobson/CCC/blob/master/gta/TempAnomalies.java

**Comment by luc4leone** [comment]
unfortunately I don't know java, but I'll try to understand what the code does, at least at a high level. thanks 😊

**Comment by bethrobson** [comment]
If I get some time, I'll do a bonus video and go through it...

**Comment by luc4leone** [comment]
>If I get some time, I'll do a bonus video and go through it...

that would be great 😉
